dropwizard-jackson/src/test/java/io/dropwizard/jackson/FuzzyEnumModuleTest.java

```diff
@@ -83,17 +101,22 @@ public void failsOnIncorrectValue() throws Exception {
                     .isEqualTo("wrong was not one of [NANOSECONDS, MICROSECONDS, MILLISECONDS, SECONDS, MINUTES, HOURS, DAYS]");
         }
     }
- 
+
     @Test
     public void mapsToLowerCaseEnums() throws Exception {
         assertThat(mapper.readValue("\"lower_case_enum\"", EnumWithLowercase.class))
                 .isEqualTo(EnumWithLowercase.lower_case_enum);
     }
-    
+
     @Test
     public void mapsMixedCaseEnums() throws Exception {
         assertThat(mapper.readValue("\"mixedCaseEnum\"", EnumWithLowercase.class))
                 .isEqualTo(EnumWithLowercase.mixedCaseEnum);
     }
-   
+
+    @Test
+    public void readsEnumsUsingToString() throws Exception {
+        mapper.configure(DeserializationFeature.READ_ENUMS_USING_TO_STRING, true);
```

Comment:
The configuration should be changed on a copy of the object mapper. Otherwise different test methods in this class might fail (order isn't guaranteed): [`ObjectMapper#copy()`](https://fasterxml.github.io/jackson-databind/javadoc/2.5/com/fasterxml/jackson/databind/ObjectMapper.html#copy())